Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/MoodHistoryFragment.java

```diff
@@ -96,5 +102,66 @@ public void onMoodHistoryUpdate(ArrayList<MoodEvent> newHistory) {
+         * @param recyclerView The RecyclerView to which ItemTouchHelper is attached to
+         * @param viewHolder The ViewHolder which is being dragged by the user
+         * @param target  The ViewHolder over which the currently active item is being dragged
+         * @return If this method returns true, ItemTouchHelper assumes viewHolder has been moved to the adapter position of target ViewHolder, else, it hasn't been moved
+         */
+        @Override
+        public boolean onMove(@NonNull RecyclerView recyclerView, @NonNull RecyclerView.ViewHolder viewHolder, @NonNull RecyclerView.ViewHolder target) {
+            return false;
+        }
+
+        /**
+         * Called when a ViewHolder is swiped by the user
+         * @param viewHolder The ViewHolder which has been swiped by the user
+         * @param direction The direction to which the ViewHolder is swiped
+         */
+        @Override
+        public void onSwiped(@NonNull RecyclerView.ViewHolder viewHolder, int direction) {
+            MoodEvent deletedMood = moodEventList.remove(viewHolder.getAdapterPosition());
+            moodEventService.deleteMoodEvent(getActivity(), deletedMood, MoodHistoryFragment.this);
+            recyclerAdapter.notifyDataSetChanged();
+        }
+
+        /**
+         * Called by ItemTouchHelper on RecyclerView's onDraw callback
+         * @param c The canvas which RecyclerView is drawing its children
+         * @param recyclerView The RecyclerView to which ItemTouchHelper is attached to
+         * @param viewHolder The ViewHolder which is being interacted by the User (or it was interacted and simply animating to its original position)
+         * @param dX Amount of horizontal displacement caused by user's action
+         * @param dY Amount of vertical displacement caused by user's action
+         * @param actionState The type of interaction on the View
+         * @param isCurrentlyActive  True if view is currently being controlled by the user or false it is animating back to its original state
+         */
+        @Override
+        public void onChildDraw(@NonNull Canvas c, @NonNull RecyclerView recyclerView, @NonNull RecyclerView.ViewHolder viewHolder, float dX, float dY, int actionState, boolean isCurrentlyActive) {
+            super.onChildDraw(c, recyclerView, viewHolder, dX, dY, actionState, isCurrentlyActive);
+
+            View itemView = viewHolder.itemView;
+
+            final ColorDrawable background = new ColorDrawable(Color.RED);
+            background.setBounds(itemView.getRight() + (int) dX, itemView.getTop(),   itemView.getRight(), itemView.getBottom());
+            background.draw(c);
+
+        }
+    };
+
+    /**
+     * Callback function that is triggered upon deleting a MoodEvent, which will update the recyclerAdapter.
+     */
+    @Override
+    public void onMoodUpdateSuccess() {
+        recyclerAdapter.notifyDataSetChanged();
```

Comment:
Like above, this line isn't needed. Although in this function you could add a toast message that says "Delete successful" or something